packages/ui-shared/src/components/PageTitle/PageTitle.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { cnCreate, Header } from '@megafon/ui-core';
+import './PageTitle.less';
+import Breadcrumbs, { Props as BreadcrumbsType } from '../Breadcrumbs/Breadcrumbs';
+
+type Props = {
+    /** Текст заголовка */
+    title: string;
+    /** Хлебные крошки */
+    breadcrumbs?: BreadcrumbsType['items'];
+    /** Текст бейджа */
+    badge?: string;
+    /** Класс для корневого элемента */
+    className?: string;
+    /** Дополнительные классы для внутренних элементов */
+    classes?: {
+        breadcrumbs?: string;
+    };
+    /** Ссылка на корневой элемент */
+    rootRef?: React.RefObject<HTMLDivElement>;
+};
+
+const cn = cnCreate('mfui-beta-page-title');
+const PageTitle: React.FC<Props> = ({ title, breadcrumbs, badge, className, classes= {}, rootRef }) => (
+    <div className={cn([className])} ref={rootRef}>
+        {breadcrumbs?.length &&
```

Comment:
обезопасим от рендера `0`

```suggestion
        {!!breadcrumbs?.length &&
```